Firely.Fhir.Packages/Source/FhirPackageSource.cs

```diff
@@ -90,14 +90,7 @@ static IEnumerable<PackageDependency> createDependencies(string[] dependencies)
             {
                 foreach (var dep in dependencies)
                 {
-                    var splitDependency = dep.Split('@');
-                    if (splitDependency.Length == 1)
-                        yield return new PackageDependency(dep, "latest");
-                    else
-                    {
-                        var versionDep = splitDependency[1];
-                        yield return new PackageDependency(splitDependency[0], versionDep);
-                    }
+                    yield return (PackageDependency)dep;
```

Comment:
Isn't this just `dependencies.Cast<PackageDependency>()` ?

Reply:
That does not work for user-defined operators sadly:
https://stackoverflow.com/questions/13316718/explicit-implicit-cast-operator-fails-when-using-linqs-cast-operator

Reply:
I don't like this cast operator magic anyway, it's basically hiding the creation of a PackageDependency object here.....why not just  `new PackageDependency(dep)` ? Everyone recognizes that....

Reply:
But ok, that's a bigger change.